semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/CallMethods.scala

```diff
@@ -2,7 +2,12 @@ package io.shiftleft.semanticcpg.language.nodemethods
 
 import io.shiftleft.codepropertygraph.generated.nodes
 import io.shiftleft.semanticcpg.language._
+import scala.jdk.CollectionConverters._
 
 class CallMethods(val node: nodes.Call) extends AnyVal {
-  def argument(index: Int): nodes.Expression = node.start.argument(index).head
+  def argumentOption(index: Int): Option[nodes.Expression] =
```

Comment:
Lets go for `Iterable` instead of `Option` as return type. That way we might return vargs later.